.github/workflows/doc-md-links.yml

```
@@ -0,0 +1,15 @@
+name: Lint doc Markdown links
+
+on:
+  push:
+    paths:
+      - 'docs/**/*'
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Prepare front-end environment
+        uses: ./.github/actions/prepare-frontend
+      - name: Lint docs
```

A nit, really. Just for the sake of consistency.

```suggestion
      - name: Lint doc links
```